docs/UserGuide.md

```diff
@@ -126,18 +126,24 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [e/EMAIL] [g/GENDER] [a/AGE] [t/ADD-TAG] [-t/DELETE-TAG]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* When editing tags:
+  * Prefix:`t/ADD-TAG]` 
```

Comment:
was having only 1 half of the square bracket ] intentional? if so, why?

Reply:
if it was unintentional, the delete tag prefix u wrote below also has only ] if u want to change that

Reply:
Oops unintentional. Will fix, thanks!